frontends/web/src/components/forms/input.tsx

```diff
@@ -15,39 +15,21 @@
  * limitations under the License.
  */
 
-import { forwardRef } from 'react';
+import { ChangeEvent, HTMLProps, forwardRef } from 'react';
 import styles from './input.module.css';
 
-type withValue = {
-  value: string | number;
-  defaultValue?: string | number;
-} | {
-  defaultValue: string | number;
-}
```

Comment:
question: this forced us to either set defaultValue OR value + optional defaultValue, but now it's gone. Was this in the way somehow?

Reply:
> IMO it's a given that input will need to have a value

fine with me, but then can/should we make value mandatory?

Reply:
ack lets keep as is for now.